.travis.yml

```diff
@@ -7,6 +7,8 @@ script:
   - yarn run test-ci
   - yarn run build
   - yarn run lint
+before_install:
+  - yarn config set ignore-engines true
```

Comment:
Temporary until https://github.com/projectfluent/fluent.js/pull/164 get's merged

Reply:
Isn't this actually a better temporary solution to this problem than projectfluent/fluent.js#164? Assuming the longer term solution is to drop the support for Node 6 in `addons-linter`.

Reply:
No, it's not since the linter is being installed by users via web-ext - https://github.com/mozilla/web-ext and a strict engine in `package.json` doesn't allow the user to install the linter via npm or yarn.